### DISCUSSION TITLE
More email address options

### DISCUSSION THREAD

**Discussion by magenta404** [community/community#3928] [Other Feature Feedback, Questions, & Ideas]
Is it possible to add more email addresses for important notifications (like a new sponsor)? I couldn't see this anywhere and it would be helpful. I am part of the organisation which receives sponsors, but I don't receive any of the emails despite correct settings in my email preferences.

We would like to also receive the sponsor _Updates_ to a designated list of email addresses for sanity checking. Right now we have to become sponsors ourselves.

**Comment by bdresser** [comment]
Hey @daleeidd — we don't yet offer super granular email permissions, but it sounds like webhooks might help with some of your usecase. You can configure them to alert you when certain events occur. Check out our documentation [here](https://docs.github.com/en/sponsors/integrating-with-github-sponsors/configuring-webhooks-for-events-in-your-sponsored-account#managing-webhooks-for-events-in-your-sponsored-account) and let me know if you have any questions!
